packages/terra-functional-testing/lib/test-runner/wdio.conf.js

```diff
@@ -0,0 +1,130 @@
+exports.config = {
```

Comment:
Can we move to config dir?
```
packages/terra-functional-testing/config/wdio.config.js
```

Reply:
I'm open to moving these files around. Is there strong reasoning for moving this file or is it to make it consistent with other existing packages?

Reply:
For me, a few reasons
1. consistency with other packages
2. lib directories make me think of compiled / ship api plus we separate our tests from lib so separating config from config makes sense to me
3. less to write when using int eh command line like: `node_modules/terra-fuctional-testing/config/wdio.config` 

I also like knowing where to find the configs instead of having to dig through nested dirs on git-hub to find a file ref. that's a personal preference 😄

Reply:
I moved the wdio.conf.js into the config directory. I might change my mind on this decision when I start integrating the selenium configurations. I'm a little conflicted about the services and configurations that support those services being disconnected in the file structure, but we can start those discussions when we get there.

Reply:
A little history here, I added the config directory because I was mad that everything in src was getting transpiled. I didn't realize you could setup babel config to ignore folders. I think i'd propose, as a compromise, to place the config folder inside the lib folder.